src/libraries/System.Net.WebSockets.Client/tests/ConnectTest.cs

```diff
@@ -313,5 +315,25 @@ public async Task ConnectAsync_CancellationRequestedAfterConnect_ThrowsOperation
                 catch (IOException) { }
             }, new LoopbackServer.Options { WebSocketEndpoint = true });
         }
+
+        [ConditionalFact(nameof(WebSocketsSupported))]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/34690", TestPlatforms.Windows, TargetFrameworkMonikers.Netcoreapp, TestRuntimes.Mono)]
+        [SkipOnPlatform(TestPlatforms.Browser, "CollectHttpResponseDetails not supported on Browser")]
+        public async Task ConnectAsync_Failed()
```

Comment:
ah, noticed it is already checked in a test above

Reply:
Still it is strange to me that it is checked in some random test about custom headers from client side. I believe it is better to have a separate test dedicated to that. And it would be also nice to pass a custom header from server in that test and check you can see it on client, not only that the collection is not empty